### ISSUE TITLE
Training 150 classes

### ISSUE THREAD

**Issue by yifita** [ShaoqingRen/faster_rcnn#62]
In the hope to expand the training for a subset of ILSVRC DET task, I expanded the dataset to with ILSVRC challenge images. The whole training dataset has 110k images for 152 classes. 

I modified the solver to be 80k (stepsize) 160k (max iteration) and reduced image scales to from 600 to 400. 

Now in the training and validation my fast-rcnn could reach 0.2 classification error (0.8 accuracy) and 0.9+ cls loss (softmax loss). But when testing with new images, my detector gives quite random output (score ~= 0.01). 
- i tested the same image using published model, works all fine. 

Any insights in why training output varies from testing result? And in general any suggestion in extending the classes?
